### PR TITLE
Create GH9.json

### PR DESCRIPTION
**Pull request by GH9youtube**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/GH9.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "GH9yt.is-a.dev",
+  "repo": "https://gh9youtube.github.io/GH9.github.io/",
+  "owner": {
+    "username": "GH9",
+    "email": "shyshyleprodu62@gmail.com"
+  },
+  "record": {
+    "CNAME": "GH9.github.io/"
```
Comment (21Z) [inline domains/GH9.json:9]:
```suggestion
    "CNAME": "GH9.github.io"
```
`CNAME` can't contain `/`

File: domains/GH9.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "GH9yt.is-a.dev",
+  "repo": "https://gh9youtube.github.io/GH9.github.io/",
+  "owner": {
+    "username": "GH9",
```
Comment (21Z) [inline domains/GH9.json:5]:
```suggestion
    "username": "GH9youtube",
```
username should be your github username

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/GH9.json (hunk 2 above)
Comment (Stef-00012) [inline domains/GH9.json:5]: You should put your github username here

File: domains/GH9.json (hunk 1 above)
Comment (Stef-00012) [inline domains/GH9.json:9]: CNAME can not contain /

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**